Hover issue is now solve

LalitDeore opened the pull request:
Issue detail.
After hovering to the "about our slack" button, it was disappearing. and i change the VButton component to the normal button and add own tailwind css to it. 

Normal Button

![normal](https://github.com/devedmonton/DES-Website/assets/92305713/fed0c6bb-0b4b-470b-b337-40e80283676a)

Hover Button

![hover](https://github.com/devedmonton/DES-Website/assets/92305713/9f7b697e-8cf1-465c-b326-7924149bd60d)

MandyMeindersma commented:
Okay so a couple things!

1. I noticed you deleted the pr template when making this pr. It is here to help you and has a couple checklist items to help you. One of the checklist items is about running certain linting commands which will make sure that you haven't broken the build. So if you want to try making a new pr and not deleting the template I think that would be a good learning opportunity. 
2. You can see the the build failed on this pr. If you click "details" the the right of `netlify/dev-edmonton/deploy-preview` then you will see that it failed because of linting things. It tell you to run prettier. 
![image](https://github.com/devedmonton/DES-Website/assets/17459171/16d0be48-d623-4291-9bf1-b168d5b2f7db)
 There are two commands that will fix this that are listed in our readme. Read through that and see if you can find them, if you get stuck, let me know


Finally:

You changed VButton to be a regular button, that isn't ideal because that isn't what the issue was asking, we only want to change the hover state. So when you are approaching this problem first I would try to investigate anything that you don't know what it is. `VButton` is actually a component that we made. If you look around the code there are actually a couple instances of it: https://github.com/search?q=repo%3Adevedmonton%2FDES-Website%20vbutton&type=code Your changes only fix VButton in one spot, we want to make sure they are fixed in all the spots. Whats the best way to do that?

MandyMeindersma reviewed (changes requested):
See comments above :)

LalitDeore commented:
Hey @MandyMeindersma First of all, thank you so much for having a detailed explanation of the problem in my PR.  

1. I am new to open source; that's why I got confused when I first saw the PR template. I will make sure to follow the PR template for the next issue.  
2. I didn't understand the second point: which command do I have to run, or what is it for? Could you please give me more details?
3. I changed my previous code because it was updating only one button. Now keep the same component of VButton and and updated css for VButton. Please review these changes and tell me if is right or if needed different change.

Hover Button:
![hoverButton](https://github.com/devedmonton/DES-Website/assets/92305713/3784ed43-2477-4cf8-9498-70322c6feb4f)

MandyMeindersma commented on components/VButton.vue:
```diff
@@ -8,7 +8,7 @@
                     : null,
                 appearance === 'inverted'
-                    ? 'text-white hover:text-brand-primary bg-brand-primary hover:bg-white'
+                    ? 'text-white hover:text-brand-primary bg-brand-primary hover:bg-white hover: hover:border-1 border-solid border-sky-500'
```
nice!! Great work

Okay couple things: 
1. I would love for the border to be our brand colour. I am thinking you should be able to do that with something like `bg-brand-primary` but I haven't tested that, so it could be wrong
2. You seem to have an extra floating `hover:` thats not attached to anything
3. Technically with tailwind they like when you add `hover:` in front of every property you add, so add a couple more :)

MandyMeindersma commented:
Yay for being new to open source! Glad you made a pr! So exciting!

just a couple more changes and then we should be good to merge!

As for the commands you have to run, you didn't break anything this build so you don't have to run them but it's good to know what all of the commands in the readme do. I was talking about these ones:

```
# run the linter
$ npm run lint-check # check for linting errors
$ npm run lint-fix # fix linting errors

# run the formatter
$ npm run format-check # check for formatting errors
$ npm run format # fix formatting errors
```

MandyMeindersma commented:
Oh also one more thing. When you click the "Details" button now you can see a test website with your changes. When I hover I don't see the border you added. I think adding more `hover:` like I suggested should fix that

![image](https://github.com/devedmonton/DES-Website/assets/17459171/de78eb32-b1a1-46e6-82b3-b374a47eb060)

MandyMeindersma commented on components/VButton.vue:
```diff
@@ -5,10 +5,10 @@
                     : null,
                 appearance === 'inverted'
-                    ? 'text-white hover:text-brand-primary bg-brand-primary hover:bg-white'
+                    ? 'text-white hover:text-brand-primary bg-brand-primary hover:bg-white  hover:border border-solid hover:border-brand-primary'
```
```suggestion
                    ? 'text-white hover:text-brand-primary bg-brand-primary hover:bg-white hover:border hover:border-solid hover:border-brand-primary'
```

MandyMeindersma commented on components/VButton.vue:
```diff
@@ -5,10 +5,10 @@
             class="inline-flex items-center px-3 py-2 border border-transparent text-sm leading-4 font-medium rounded-md text-white border-brand-primary focus:outline-none focus:border-indigo-700 focus:shadow-outline-indigo active:bg-indigo-700 transition ease-in-out duration-150"
             :class="[
                 appearance === 'default'
-                    ? 'text-brand-primary hover:text-white bg-white hover:bg-brand-primary hover:text-white'
```
why did you remove this one?

MandyMeindersma commented:
Oh the joys of dealing with merge conflicts now too! You really are getting the full experience of open source. Let me know if you need any help with the merge conflicts

LalitDeore commented:
Yes, It will be helpful if you tell me how I can solve the merge conflict issue.

MandyMeindersma commented:
Try doing some googling or following this tutorial https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-using-the-command-line

If you still have questions after that let me know :)

LalitDeore commented:
> dealing with merge conflicts

Is this issue resolved? I have solved the merge conflict with the main branch. Please checkout once.

MandyMeindersma commented:
Almost! We are so close! 

You just have one other suggestion left from me. If you go to the "files changed" tab it should show. I just need one more `hover:`

LalitDeore commented:
Added one more hover

MandyMeindersma commented:
Looks Great!

MandyMeindersma commented:
@all-contributors add @LalitDeore for code
